_data/previous-toc.yml

```diff
@@ -0,0 +1,23 @@
+- title: Documentation
```

Comment:
Should this file be deleted now?

Reply:
Likewise that was removed from my commit! Let me check if I messed something up.

Reply:
The last commit should have sorted this out. Thanks